cli/azd/pkg/tools/python.go

```diff
@@ -19,8 +20,34 @@ func NewPythonCli() *PythonCli {
 	return &PythonCli{}
 }
 
-func (cli *PythonCli) CheckInstalled(_ context.Context) (bool, error) {
-	return toolInPath(pythonExe())
+func (cli *PythonCli) versionInfo() VersionInfo {
+	return VersionInfo{
+		MinimumVersion: semver.Version{
+			Major: 3,
+			Minor: 7,
+			Patch: 6},
+		UpdateCommand: "Visit https://www.python.org/downloads/ to upgrade",
+	}
+}
+
+func (cli *PythonCli) CheckInstalled(ctx context.Context) (bool, error) {
+	found, err := toolInPath(pythonExe())
+	if !found {
+		return false, err
+	}
+	pythonRes, err := executeCommand(ctx, "python3", "--version")
```

Comment:
```suggestion
	pythonRes, err := executeCommand(ctx, pythonExe()", "--version")
```

Reply:
We should use `pythonExe()` here as we do above because the executable name differs between windows and non windows.